### PR TITLE
Update windows-local-autopilot-reset.md

### PR DESCRIPTION
**Pull request by msjbja**
Addition of the permission required to perform this task

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR based on issue #2651. Thanks.

File: windows/deployment/windows-autopilot/windows-autopilot-reset-local.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ ms.topic: article
 
 **Applies to: Windows 10, version 1709 and above
 
+The Microsoft Intune Service Administrator Directory role is required to perform this task. The process of adding directory roles is documented at https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/windows-autopilot-reset-local.md]:
```suggestion
The Microsoft Intune Service Administrator Directory role is required to perform this task. To learn more about the process of adding directory roles, refer to https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal
```

Comment (nenonix) [inline windows/deployment/windows-autopilot/windows-autopilot-reset-local.md]:
```suggestion
The Intune Service Administrator role is required to perform this task.  Learn more about how to [Assign Azure Active Directory roles](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal).
```

File: windows/deployment/windows-autopilot/windows-autopilot-reset-local.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ ms.topic: article
 
 **Applies to: Windows 10, version 1709 and above
 
+The Microsoft Intune Service Administrator Directory role is required to perform this task. To learn more about the process of adding directory roles, refer to https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal
```
Comment (mypil) [inline windows/deployment/windows-autopilot/windows-autopilot-reset-local.md]:
```suggestion
The Intune Service Administrator role is required to perform this task.  Learn more about how to [Assign Azure Active Directory roles](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-users-assign-role-azure-portal).
```

**Comment by mypil** [comment]
@greg-lindsay - Can you please assist to check, approve and merge the content for this PR based on issue #2651?

Thank you.

**Comment by mypil** [comment]
@greg-lindsay - We would like to follow up on this.  Can you please assist to check, approve and merge the content for this PR based on issue #2651?

Thank you.

**Comment by mypil** [comment]
@greg-lindsay @AndreaBarr - Would you be able to help merge the content of this PR? Thanks.

**Comment by mypil** [comment]
@Justinha @DaniHalfin @mjcaparas @lizap @greg-lindsay @AndreaBarr - Would you be able to help merge the content of this PR? Thanks.

**Comment by mypil** [comment]
@greg-lindsay - We would like to follow up. Can you please assist to check, approve and merge the content of this PR?

Thank you.

**Pull request merged**